Formula/git-credential-libsecret.rb

```diff
@@ -0,0 +1,29 @@
+class GitCredentialLibsecret < Formula
+  desc "Git helper for accessing credentials via libsecret"
+  homepage "https://git-scm.com"
+  # NOTE: Please keep these values in sync with git.rb when updating.
+  url "https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.33.0.tar.xz"
+  sha256 "bf3c6ab5f82e072aad4768f647cfb1ef60aece39855f83f080f9c0222dd20c4f"
+  license "GPL-2.0-or-later"
+  head "https://github.com/git/git.git", branch: "master"
+
+  livecheck do
+    formula "git"
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on "libsecret"
+
+  def install
+    cd "contrib/credential/libsecret" do
+      system "make"
+      bin.install "git-credential-libsecret"
+    end
+  end
+
+  test do
+    output = shell_output("#{bin}/git-credential-libsecret 2>&1", 1)
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.